### PR TITLE
Restructure

### PR DESCRIPTION
**Pull request by sixhobbits**
* Rename categories
* Make urls consistent
* Sort articles into better categories
* Add redirects to prevent breaking external links

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -58,17 +58,218 @@ const renderMarkdown = async (path) => {
 	return marked(raw);
 }
 
+app.get('/misc/General-FAQ', async (req, res) => {
+    res.redirect('/getting-started/faq');
```
Comment (masad-frost) [inline index.js:62]: Should we set these to be perma redirects? It's better for SEO IIRC

```suggestion
    res.redirect(301, '/getting-started/faq');
```

**Review by masad-frost — APPROVED** [review]
LGTM

A couple of notes:
1. We should probably make the redirects perma-redirects for SEO, but we should make sure not to re-use any of the old links in the future since the redirect gets cached on browsers.
2. Looks like the search still points at old links, do we need to regenerate the search again?

File: getting-started/managing-repls.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Managing Your Repls
+# Managing your repls
```
Comment (jeremypress) [inline getting-started/managing-repls.md:1]: I think we always capitalize "Repls"?

File: programming-ide/getting-repl-metadata.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Getting Repl Metadata
+# Getting repl metadata
```
Comment (jeremypress) [inline programming-ide/getting-repl-metadata.md:1]: if so, update here as well

**Review by jeremypress — APPROVED** [review]
+1 to Faris' comment, and a question of if we should capitalize "Repl" like we do for Replit. Technically it's an acronym (REPL) but the product feature has diverged from the original name

**Comment by masad-frost** [comment]
We use lowercase `repl`, see https://replit.com/@util/handbook#support/style.md

**Comment by jeremypress** [comment]
> We use lowercase `repl`, see https://replit.com/@util/handbook#support/style.md

ah nice, thanks

File: index.js (hunk 1 above)
Comment (sixhobbits) [reply index.js]: done, thanks for catching

**Comment by sixhobbits** [comment]
> We use lowercase `repl`, see https://replit.com/@util/handbook#support/style.md

@masad-frost we didn't know you had a style guide! Would it be possible for us to get access to that? We'll make sure the Repl/repl thing is consistent.

**Pull request merged**